places-solution-integrations/campaign/postbacks.md

```diff
@@ -0,0 +1,76 @@
+# Sending Places Data to Campaign via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to send Places data to your Adobe Campaign servers. With your desired property selected in Launch, you can create this type of rule by completing four steps:
```

Comment:
Change to:
"After Places sends entry and exit events, you can leverage Rules in Launch to send Places data to your Adobe Campaign servers. To create this rule, select your property in Launch, and complete the following steps: